apps/platform-app/src/components/LanguageInput/LanguageInput.tsx

```diff
@@ -0,0 +1,190 @@
+import { type ProfileFormValues } from '@/app/[locale]/(profile)/profile.types'
+import CancelIcon from '@/assets/icons/CancelIcon'
+import ImportantIcon from '@/assets/icons/ImportantIcon'
+
+import { type DropdownOption } from '@/components/Dropdowns/DropdownOptionItem/DropdownOptionItem'
+import styles from '@/components/TechStackInput/TechStackInput.module.scss'
+import languages from '@/data/languages'
+import classNames from 'classnames/bind'
+import { useFormikContext } from 'formik'
+import React, { useEffect, useRef, useState } from 'react'
+import Tooltip from '../Tooltip/Tooltip'
+
+const cx = classNames.bind(styles)
+
+interface LanguageInputProps {
+  chips: DropdownOption[]
+  onTechSelect: (lang: DropdownOption) => void
+  onTechRemove: (lang: DropdownOption) => void
+  placeholder: string
+  name: string
+  label: string
+  addImportantIcon?: boolean
+  tooltipText?: string | null
+}
+
+export const mapLanguagesToDropdownOptions = (languages: string[]) => {
+  return languages.map((lang) => ({
+    name: lang,
+    value: lang,
+  }))
+}
+
+const initialLanguage: DropdownOption = { name: '', value: '' }
+
+export const LanguageInput: React.FC<LanguageInputProps> = ({
+  chips = [],
+  onTechSelect,
+  onTechRemove,
+  placeholder,
+  name,
+  label,
+  addImportantIcon,
+  tooltipText,
+}) => {
+  const { errors, handleBlur, touched } = useFormikContext<ProfileFormValues>()
```

Comment:
**nit:** We should place this component close to the form there its used. It's because it uses `useFormikContext`, and it will be impossible to move it later to `ui-system`